readme.md

```diff
@@ -0,0 +1,34 @@
+# Fstat [![Build Status](https://travis-ci.org/gluster/fstat.svg?branch=master)](https://travis-ci.org/gluster/fstat)
+Fstat tracks the failures from build.gluster.org.
+
+## Installation
+- Before you start installation make sure you have working installation of python2
+- In your virtualenv run ``` pip install -r requirements.txt ```
+- Hurray the installation is done
+
+## Using fstat
+- To serve the fstat app on localhost run ``` python manage.py runserver ```
```

Comment:
This will most likely fail without configuring a database with `python manage.py db upgrade` at the start.